### ISSUE TITLE
Add numbers to separate files with the same name

### ISSUE THREAD

**Issue by yukina3230** [jely2002/youtube-dl-gui#215]
When downloading, if the video has the same name as the file in the download folder, it will be overwritten by the old file (When I clicked at the "Open file" button, it's open the old video, the file I just downloaded doesn't appear in the download folder)

**Comment by jely2002** [comment]
I'll close this for now as there is no way to currently do this with yt-dlp, see; https://github.com/yt-dlp/yt-dlp/issues/5485.
If I do implement this it would be an action performed by the GUI, issue with this is that I can only do something before or after yt-dlp finishes. I can't know the output file before, as it can depend on a lot of template variables. After the download it has already been downloaded, and it's already been overwritten.

TL;DR this issue should be fixed upstream in yt-dlp, its technically complex to add a feature like this to the GUI itself.
